pkgs/tools/security/dkimpy/default.nix

```diff
@@ -0,0 +1,44 @@
+{ stdenv, pythonPackages, fetchurl, openssl, makeWrapper }:
```

Comment:
@leenaars yes, if it is both an application and library, move it to python-modules and add an top-level alias to all-packages.nix from pythonPackages.